backend/app/views/spree/admin/return_authorizations/_form.html.erb

```diff
@@ -76,13 +76,13 @@
   <% end %>
 
   <%= f.field_container :stock_location do %>
-    <%= f.label :stock_location, Spree.t(:stock_location) %>
+    <%= f.label :stock_location_id %>
```

Comment:
Yeah.  This wasn't merged in yet but it would be taken care of https://github.com/solidusio/solidus/pull/876/files#diff-3 if it's merged first or need to be rebased to take care of.  I did this wrong in a number of places (every place in that PR) but since some were merged and some not I thought it made the most sense to handle it all in one PR and rebase as necessary.